src/models/settings.ts

```diff
@@ -66,4 +67,5 @@ export const DEFAULT_SETTINGS: Settings = {
   trackInfoBackgroundOpacity: 50,
   showFreemiumWarning: true,
   cornerRadius: 0,
+  refreshTime: 1000,
```

Comment:
Actually, you can just store in seconds... so this would become (note the variable name)

```suggestion
  trackInfoRefreshTimeInSeconds: 1,
```